deploy/docker/cp-dts-status-checker/dts-status-checker.py

```diff
@@ -0,0 +1,139 @@
+from functools import total_ordering
+import json
+import subprocess
+import os
+import sys
+from pipeline.api import PipelineAPI
+from pipeline.log.logger import LocalLogger, TaskLogger, RunLogger
+import codecs
+import copy
+
+FS_ROOT = "/"
+CLOUD_DATA_DIR = "cloud-data"
+DTS_STATUS_FILE = "data.json"
+DTS_EMAIL_TEMPLATE = "template.html"
+
+dts_user = os.environ.get('CP_DTS_STATUS_NOTIFICATION_USER')
+if dts_user is None:
+    raise RuntimeError("CP_DTS_STATUS_NOTIFICATION_USER is not defined!")
+dts_users_copy = os.getenv('CP_DTS_STATUS_NOTIFICATION_USERS_COPY')
+dts_users_copy_list = dts_users_copy.replace(" ", "").split(',')
+dts_notify_subject = os.environ.get(
+    'CP_DTS_STATUS_NOTIFICATION_SUBJECT', 'DTS statuses were changed')
+dts_path = os.getenv('CP_DTS_STATUS_STORAGE')
```

Comment:
regarding to your .md file `dts_path` is a required parameter, so lets check that this parameter is configured and fail if it is not, as it done for dts_user. Also we can check that full path `os.path.join(FS_ROOT, CLOUD_DATA_DIR, dts_path)`  exists and it is folder

Reply:
+ as discussed, now this variable should represent full path to local directory, e.g. `/cloud-data/dts-status-checker-storage`, so lets also check that this path exists and it is directory, before proceed with a script